popupwindow.js

```diff
@@ -0,0 +1,121 @@
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Multi-post <br/>Stories",
+    image: "./imagsPop/multi1.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Facebook",
+    image: "./imagsPop/facebook1.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Multi-post <br/>Stories",
+    image: "./imagsPop/project image 2.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+];
+
+const modal = document.querySelector(".modal");
+const popupWrap = document.querySelector(".pop-up-wrap");
+const projectBtn = document.querySelectorAll(".see-btn");
+
+projectBtn.forEach(function (Btn) {
+  Btn.addEventListener("click", function (e) {
+    const i = e.currentTarget.dataset.id;
+    popupWrap.innerHTML = `<div class="pop-up-nav">
+		  <h2 class="poppins" id="logo">${dataPopover[i].heading}</h2>
+		  <span class = "pop-up-closeBtn">&times;</span>
+		</div>
+	  
+	  <div class="pop-up-about">
+		  <ul class="poppins canopy-modal">
+			<li class="about-li" id="canopy">CANOPY</li>
+			<li class="about-li">Back End Dev</li>
+			<li class="about-li">2015</li>
+		  </ul>
+	  </div>
+	  
+	  <div class="pop-up-image-wrap">
+		<img class="snapshoot" src="${dataPopover[i].image}" alt="Tanner Christensen's portfolio image">
+	  </div>
+		 
+	  <div class="pop-up-text-content">
+		<p class="poppins-regular" id="modal-text">${dataPopover[i].content}</p>
+		<div class="tag-wrap">
+		<ul class="poppins-regular language-button">
+		  <li class="languages">html</li>
+		  <li class="languages">css</li>
+		  <li class="languages">javascript</li>
+		</ul>
+	  <div class="pop-up-btn">
+		<button class="poppins-regular see-live-btn">See live<img class="live-icon" src="./imagsPop/see-live-icon.svg"/></button>
+		<button class="poppins-regular see-source-btn">See source<img id="github-modal-icon" src="./images/Vector(2).svg"/></button>
```

Comment:
- [ ] it would be better if you use the same background color of the buttons in the Figma design in yours in order to match the design 

- [ ] also you should add the divider line in both versions of mobile and desktop in order to match the Figma design 

- Yours 
<img width="279" alt="Screen Shot 2022-10-19 at 17 59 10" src="https://user-images.githubusercontent.com/61790587/196757021-100b01ad-0d10-4546-a6cf-9877af760aff.png">
- Figma 
<img width="278" alt="Screen Shot 2022-10-19 at 17 59 55" src="https://user-images.githubusercontent.com/61790587/196757045-c9dd34de-8f6d-4612-a634-cf5903f28834.png">

Reply:
Kindly, I suggest you follow the advice of the previous reviewer and adjust the button layout to look more like the original ones in Figma.

Reply:
I change the color of the text and background text.

Reply:
I changed the background color